.yamato/package-test.yml

```diff
@@ -20,7 +21,12 @@ test_{{ test.name }}_{{ platform.name }}_{{ editor }}:
   {% endif %}
   commands:
     - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
-    - upm-ci package test -u {{ editor }} {{ test.args }}
```

Comment:
promotion-audit test is not a valid argument for project test, also, the file is still named package-test. I think you might need SRPs tests to be their own job, so that you don't have to touch the structure of the jobs so much?